test/tests/geomsearch/1d_line_sampler/tests

```diff
@@ -1,7 +0,0 @@
-[Tests]
-  [./test]
-    type = 'CSVDiff'
-    input = '1d_line_sampler.i'
-    csvdiff = '1d_line_sampler_out_my_vpp_0001.csv'
```

Comment:
If you delete this test, you also need to delete the gold files and input files.

Reply:
Oh I see, it's a rename... NVM.